### DISCUSSION TITLE
PODS CrashLoopBackOff

### DISCUSSION THREAD

**Discussion by manoj199x** [egovernments/Digit-Core#262] [DIGIT Platform]
Hi @team i was trying to deploy the digit quick setup  in AWS.. but some pods are not running, constantly failing to init.
<img width="1512" alt="Screenshot 2023-10-11 at 3 25 49 PM" src="https://github.com/egovernments/Digit-Core/assets/37971143/37afc703-fc4c-4ca8-a5a1-ab128dccde48">

**Comment by manoj199x** [comment]
Describe pods returned these
<img width="1512" alt="Screenshot 2023-10-11 at 3 35 18 PM" src="https://github.com/egovernments/Digit-Core/assets/37971143/9750261f-25d4-4ada-bdd7-bed8f10915fe">
<img width="1512" alt="Screenshot 2023-10-11 at 3 35 35 PM" src="https://github.com/egovernments/Digit-Core/assets/37971143/280ac2f3-00a3-4f97-97a8-45f0ef9b54af">
<img width="1512" alt="Screenshot 2023-10-11 at 3 35 28 PM" src="https://github.com/egovernments/Digit-Core/assets/37971143/7433e13b-c8c8-4667-a9f4-bafcf9bbccc5">

**Reply by Ros-web** [reply]
Hi @manoj199x - Got any solution for this  i am also facing the  same issues.

![image](https://github.com/egovernments/Digit-Core/assets/60926594/f3255e97-af61-4dad-b89e-bbe4f205d3d0)

**Comment by nikhilmulinti** [comment]
Please share kubectl describe <pod_name> -n egov output of the pod which is in crashloopbackoff state

**Comment by alhamdubello** [comment]
I am also experiencing the same issues please any idea for a solution? 
<img width="1057" alt="Screenshot 2024-02-16 at 12 26 43" src="https://github.com/egovernments/Digit-Core/assets/8626582/c005140f-017c-4d64-ad47-40b3c5742ed8">

**Comment by alhamdubello** [comment]
I also posting my kubectl describe pod egov-user-665f8dff69-vtq7x -n egov
![Screenshot 2024-02-16 at 12 29 16](https://github.com/egovernments/Digit-Core/assets/8626582/7c80e575-6a07-4ed2-a898-c5ca7f8fe78a)
![Screenshot 2024-02-16 at 12 29 03](https://github.com/egovernments/Digit-Core/assets/8626582/0606810a-9e9a-4a0d-a294-4c84b9b8e4e1)

**Reply by nikhilmulinti** [reply]
Hi @alhamdubello Please let us know which guide you are referring to install DIGIT
